appinventor/appengine/war/static/css/Ya.css

```diff
@@ -1517,9 +1522,9 @@ path.ode-SimpleMockMapFeature-selected {
 }
 
 .ode-ComponentHelpPopup-Body {
-  padding: 10px;
```

Comment:
Note that removing this causes the content of the body to not line up with the "More Information" link.